CriteoPublisherSdk/Sources/Util/CR_URLResolver.m

```diff
@@ -69,11 +69,24 @@ - (void)resolverURL:(NSURL *)url
          deviceInfo:(CR_DeviceInfo *)deviceInfo
          resolution:(CR_URLResolutionHandler)resolutionHandler {
   self.resolutionHandler = resolutionHandler;
+  if ([self resolveURL:url]) {
+    return;
+  }
+
   CR_URLRequest *request = [CR_URLRequest requestWithURL:url deviceInfo:deviceInfo];
   NSURLSessionDataTask *task = [self.urlSession dataTaskWithRequest:request];
   [task resume];
 }
 
+- (BOOL)resolveURL:(NSURL *)URL {
```

Comment:
I don't understand the naming of this method and I'm confused when reading the code because of the name.

It's trying to resolve AppStore URL if it is an actual AppStore URL.
What do you think about something like `tryResolveAppStoreURL` ?

Reply:
Fixed